test/m1_facilitator/repositories/GatewayRepository/util.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+
+
+import assert from '../../../test_utils/assert';
+import Gateway from '../../../../src/m1_facilitator/models/Gateway';
+
+const Util = {
+
+  assertGatewayAttributes(
+    inputGateway: Gateway,
+    expectedGateway: Gateway,
+  ): void {
+    assert.strictEqual(
+      inputGateway.gatewayGA,
+      expectedGateway.gatewayGA,
+      'Mismatch in gateway\'s global address.',
+    );
+
+    assert.strictEqual(
+      inputGateway.remoteGA,
+      expectedGateway.remoteGA,
+      'Mismatch in remote\'s global address',
```

Comment:
What is exactly a remote global address?

Reply:
Changed it to `Mismatch in remote gateway global address.`